pages/api/senado/v1/gestao/servidores/aposentadoria/inativos.js

```diff
@@ -0,0 +1,21 @@
+import app from '@/app';
+import BadRequestError from '@/errors/BadRequestError';
+import NotFoundError from '@/errors/NotFoundError';
+import { getDadosServidoresInativos } from '@/services/dados-abertos-br/senado-gestao-pessoas';
+
+async function getDadosGestaoInativos(request, response) {
+  try {
+    const result = await getDadosServidoresInativos();
+    return response.status(result.status).json(result.data);
```

Comment:
Creio que qualquer status que não seja `ok ` cairá no catch, então aqui podemos fixar o status 200
```suggestion
    return response.status(200).json(result.data);
```